src/job_cache/db_helpers.h

```diff
@@ -0,0 +1,247 @@
+/*
```

Comment:
These were all just moved out of job_cache.cpp so that they could be used across the job_cache folder.